apps/salesforce-commerce-cloud/package.json

```diff
@@ -14,8 +15,8 @@
     "@tanstack/react-query": "^4.29.7",
     "@vitejs/plugin-react": "^4.3.1",
     "axios": "^1.7.7",
-    "commerce-sdk": "^2.10.1",
-    "contentful-management": "10.7.0",
+    "commerce-sdk": "^2.15.0",
+    "contentful-management": "^11.35.1",
```

Comment:
might cause a problem again, we'll see

Reply:
Curious do we know what specific problem did this library update cause in the past? I think I've merged in a few with the update to 11 and they were still working in prod. 

I am working on getting staging setup for this one. I got a staging environment set up and deployed the bundle but ran out of time to go through the config. I'll make sure that's done and tested before I merge.

Reply:
I can also update this to the earliest version with a fix (assuming that's even an option) if it's safer.